benchopt/utils/dependencies_mixin.py

```diff
@@ -99,8 +99,20 @@ def install(cls, env_name=None, force=False):
             try:
                 cls._pre_install_hook(env_name=env_name)
                 if cls.install_cmd == 'conda':
-                    install_in_conda_env(*cls.requirements, env_name=env_name,
-                                         force=force)
+                    try:
+                        install_in_conda_env(*cls.requirements,
+                                             env_name=env_name,
+                                             force=force)
+                    except AttributeError:
```

Comment:
there could be an AttributeError due to another attribute than `requirements` missing. Instead of try except, should we use `if hasattr(cls, "requirement") / else` ?

Reply:
Yes this would be better

Reply:
I pushed a new commit. Tell me if I need to modify again the file